src/sonic-yang-models/yang-models/sonic-acl.yang

```diff
@@ -0,0 +1,368 @@
+/* this is sonic py yang model */
```

Comment:
Please remove this file. It is mistakenly added. ACL is part of template folder

Reply:
All 4 files that were added by mistake have been removed.